CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.0]
+
+### Removed
+* Support for Python 3.8 and 3.9 has been dropped, 3.10 is now the minimum version.
+
+
+## [3.0.1]
+
+### Removed
+* The `six` package has been dropped as a dependency as it's no longer used since we don't support Python 2 anymore.
+
```

Comment:
```suggestion
## [3.1.0]

### Removed
* Support for Python 3.8 and 3.9 has been dropped, 3.10 is now the minimum version.
* The `six` package has been dropped as a dependency as it's no longer used since we don't support Python 2 anymore.

```